build.gradle

```diff
@@ -128,7 +128,7 @@ jacocoTestReport {
     }
 }
 
-String version = '8.2.1'
+String version = '8.2.2'
```

Comment:
Hi @peterzhuamazon
This is a patch and new libraries went in 8.x which will need a minor release. See https://github.com/opensearch-project/opensearch-build-libraries/commits/8.x/

Please create a 8.2 branch in order to accommodate this patch or wait for next minor release. Version bump in this PR: https://github.com/opensearch-project/opensearch-build-libraries/pull/590/files but more changes are expected. 

Thanks!

Reply:
This is a patch, not even adding any new lib but rather update existing lib.

Thanks.